go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/jackc/pgconn v1.12.0
 	github.com/jackc/pgerrcode v0.0.0-20201024163028-a0d42d470451
 	github.com/jessevdk/go-flags v1.4.0
+	github.com/juju/errors v0.0.0-20220331221717-b38fca44723b
```

Comment:
@Roasbeef I'm not sure if Reviewable is incorrect or I'm using it wrong: But it was hiding this change from me ("there are no changes in these files or you don't need to review them"), which is quite weird... Going to stay with GitHub for now I think.